ocamltest/ocaml_actions.ml

```diff
@@ -81,7 +81,14 @@ let backend_flags env =
     Ocaml_variables.ocamlc_flags
     Ocaml_variables.ocamlopt_flags
 
-let dumb_term = [|"TERM=dumb"|]
+let ocaml_env =
+  let open Clflags in
+  let error_style =
+    Printf.sprintf "%s=%s"
+      error_style_reader.env_var
+      (error_style_reader.print Misc.Error_style.default_setting)
+  in
+  [| "TERM=dumb"; error_style |]
```

Comment:
I would generalize this logic to deal with `Error_style` and `Color` uniformly.

```ocaml
let env_setting env_reader default_setting = ...
let ocaml_env = [|
  "TERM=dumb";
  env_setting Clflags.error_style_reader Misc.Error_style.default_setting;
  env_setting Clflags.color_reader Misc.Color.default_setting
|]
```

Today I think that this is unnecessary because TERM=dumb implies the default color setting, but this shape of code (being careful to list all those touchy environment-decided options) scales better to the potential addition of new readers in the future.

Reply:
Also I think that `default_ocaml_env` would be a better variable name.

Reply:
Indeed, you're completely right. I just implemented your suggestion.